### ISSUE TITLE
mobs_killed track

### ISSUE THREAD

**Issue by danechek** [Auxilor/StatTrackers#4]
Please, add more separate tracks for killed monsters and killed non-hostile mobs

mobs_killed
monsters_killed
nonhostile_mobs_killed

**Comment by danechek** [comment]
do these tracks (and bosses_killed) support mobs from plugins like MythicMobs?

**Comment by fnrcum** [comment]
I agree here, an integration with MythicMobs would be great to define what MM is a boss and which is not
